### PR TITLE
removed(styles): removed styles module

### PR DESCRIPTION
**Pull request by xxxrenegade**
fixes https://github.com/webSPELL/webSPELL-4.2.3/issues/133
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23issuecomment-69507909%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23issuecomment-69510172%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22768952%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22768958%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22769882%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22770184%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22770844%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22770845%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23issuecomment-69624449%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23issuecomment-69646238%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23issuecomment-70306111%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23issuecomment-70369641%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23issuecomment-70373254%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23issuecomment-69507909%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Ist%20das%20mit%20dem%20Installer%20eigentlich%20so%20OK%2C%20den%20bei%20einem%20Update%20geht%20dann%20halt%20der%20Titel%20verloren.%22%2C%20%22created_at%22%3A%20%222015-01-11T19%3A41%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22Ich%20denke%20so%20sollte%20es%20jetzt%20passen%20%3B-%29%22%2C%20%22created_at%22%3A%20%222015-01-11T20%3A35%3A15Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22Jetzt%20sollte%20der%20installer%20richtig%20sein%20%3B-%29%22%2C%20%22created_at%22%3A%20%222015-01-12T18%3A59%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22Jetzt%20richtig%3F%22%2C%20%22created_at%22%3A%20%222015-01-12T21%3A08%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22So%20habe%20jetzt%20die%20table-striped%20Klasse%20bei%20den%20Tabellen%20hinzugef%5Cu00fcgt%2C%20ist%20sonst%20noch%20etwas%20was%20gemacht%20werden%20muss%3F%20Oder%20kann%20es%20jetzt%20zusammengef%5Cu00fchrt%20werden%3F%22%2C%20%22created_at%22%3A%20%222015-01-16T19%3A16%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22Hatten%20am%20Dienstag%20besprochen%20das%20wir%20die%20Style%20Seite%20im%20ACP%20doch%20behalten%20wollen.%20Es%20soll%20damit%20dann%20m%5Cu00f6glich%20sein%20die%20BS3%20Settings%20zu%20setzen.%5Cr%5Cn%5Cr%5CnDie%20wichtigsten%20Farb%5Cu00e4nderungen%20sollten%20damit%20m%5Cu00f6glich%20sein.%20http%3A//getbootstrap.com/customize/%5Cr%5Cn%5Cr%5CnWird%20k%5Cu00f6nnen%20aber%20hoffentlich%20teile%20deines%20PRs%20nutzen%20%28l%5Cu00f6schen%20der%20bg1-4%20variablen%20u.A.%29.%22%2C%20%22created_at%22%3A%20%222015-01-17T14%3A51%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22Alles%20klar%2C%20dann%20konzentriere%20ich%20mich%20mal%20weiter%20auf%20die%20Anpassung%20der%20Einzelnen%20Module.%22%2C%20%22created_at%22%3A%20%222015-01-17T16%3A33%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20fb7b385a9a473fd9d4c7b5d55fb51ea19ef706d3%20checklogin.php%207%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22768952%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%27t%20this%20be%20replaced%20by%20%60%3Ctable%20class%3D%5C%22table%5C%22%3E%60%3F%22%2C%20%22created_at%22%3A%20%222015-01-11T19%3A02%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-01-11T22%3A31%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20checklogin.php%3AL197-204%22%7D%2C%20%22Pull%20f221b0bb6415d56c31864e2f155399d9b07c72f2%20install/functions.php%2069%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22769882%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Das%20ist%20nicht%20ganz%20korrekt%20so.%5Cr%5CnDu%20darfst%20die%20Spalte%20nicht%20bei%20der%20Basis-Installation%20hinzuf%5Cu00fcgen%20sondern%20nur%20in%20der%20Update-Funktion%20und%20musst%20dann%20auch%20den%20Wert%20aus%20der%20Settings%20%5Cu00fcbertragen.%20Sonst%20verlieren%20alle%20die%20ein%20Update%20machen%20den%20Page-Title%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222015-01-11T20%3A37%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22Das%20ganze%20sollte%20auch%20eigentlich%20so%20lauten%3A%5Cr%5Cnmysqli_query%28%24_database%2C%20%5C%22DROP%20TABLE%20%60%5C%22%20.%20PREFIX%20.%20%5C%22styles%60%5C%22%29%3B%5Cr%5Cn%5Cr%5CnAllerdings%20sollte%20man%20vor%20dem%20l%5Cu00f6schen%20%20vielleicht%20den%20Titel%20noch%20auslesen%20und%20in%20die%20title%20spalte%20der%20settings%20tabelle%20einf%5Cu00fcgen.%20Oder%20kann%20man%20per%20SQL%20auch%20irgend%20wie%20Tabellen%20Spalten%20in%20andere%20Tabellen%20verschieben.%20Ich%20glaube%20aber%20nicht.%22%2C%20%22created_at%22%3A%20%222015-01-11T21%3A19%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20install/functions.php%3AL2409-2417%22%7D%2C%20%22Pull%20fb7b385a9a473fd9d4c7b5d55fb51ea19ef706d3%20calendar.php%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/136%23discussion_r22768958%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22No%20differentiation%20anymore...%3F%22%2C%20%22created_at%22%3A%20%222015-01-11T19%3A03%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-01-11T22%3A31%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20calendar.php%3AL221-235%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/136#issuecomment-69507909'>General Comment</a></b>
- <a href='https://github.com/xxxrenegade'></a> Ist das mit dem Installer eigentlich so OK, den bei einem Update geht dann halt der Titel verloren.
- <a href='https://github.com/xxxrenegade'></a> Ich denke so sollte es jetzt passen ;-)
- <a href='https://github.com/xxxrenegade'></a> Jetzt sollte der installer richtig sein ;-)
- <a href='https://github.com/xxxrenegade'></a> Jetzt richtig?
- <a href='https://github.com/xxxrenegade'></a> So habe jetzt die table-striped Klasse bei den Tabellen hinzugefügt, ist sonst noch etwas was gemacht werden muss? Oder kann es jetzt zusammengeführt werden?
- <a href='https://github.com/Pascalmh'></a> Hatten am Dienstag besprochen das wir die Style Seite im ACP doch behalten wollen. Es soll damit dann möglich sein die BS3 Settings zu setzen.
Die wichtigsten Farbänderungen sollten damit möglich sein. http://getbootstrap.com/customize/
Wird können aber hoffentlich teile deines PRs nutzen (löschen der bg1-4 variablen u.A.).
- <a href='https://github.com/xxxrenegade'></a> Alles klar, dann konzentriere ich mich mal weiter auf die Anpassung der Einzelnen Module.
- [x] <a href='#crh-comment-Pull f221b0bb6415d56c31864e2f155399d9b07c72f2 install/functions.php 69'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/136#discussion_r22769882'>File: install/functions.php:L2409-2417</a></b>
- <a href='https://github.com/Phhere'></a> Das ist nicht ganz korrekt so.
Du darfst die Spalte nicht bei der Basis-Installation hinzufügen sondern nur in der Update-Funktion und musst dann auch den Wert aus der Settings übertragen. Sonst verlieren alle die ein Update machen den Page-Title
- <a href='https://github.com/xxxrenegade'></a> Das ganze sollte auch eigentlich so lauten:
mysqli_query($_database, "DROP TABLE `" . PREFIX . "styles`");
Allerdings sollte man vor dem löschen  vielleicht den Titel noch auslesen und in die title spalte der settings tabelle einfügen. Oder kann man per SQL auch irgend wie Tabellen Spalten in andere Tabellen verschieben. Ich glaube aber nicht.
- [x] <a href='#crh-comment-Pull fb7b385a9a473fd9d4c7b5d55fb51ea19ef706d3 checklogin.php 7'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/136#discussion_r22768952'>File: checklogin.php:L197-204</a></b>
- <a href='https://github.com/Pascalmh'></a> Can't this be replaced by `<table class="table">`?
- [x] <a href='#crh-comment-Pull fb7b385a9a473fd9d4c7b5d55fb51ea19ef706d3 calendar.php 5'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/136#discussion_r22768958'>File: calendar.php:L221-235</a></b>
- <a href='https://github.com/Pascalmh'></a> No differentiation anymore...?

<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/136?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/136?approve=1'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/136'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by derchrisuk** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/136'>Refresh</a>.<a href='#crh-update-none'></a>

File: checklogin.php [hunk 1]
```diff
@@ -197,9 +197,7 @@
     <table class="table">
         <tr>
             <td height="500" class="text-center">
-                <table width="350" border="0" cellpadding="10" cellspacing="0" style="border:1px solid <?php
-                    echo BORDER; ?>" bgcolor="<?php
-                        echo BG_1; ?>">
+                <table width="350" border="0" cellpadding="10" cellspacing="0">
```
Comment (Pascalmh) [inline checklogin.php]: Can't this be replaced by `<table class="table">`?

File: calendar.php [hunk 2]
```diff
@@ -221,14 +221,14 @@ function print_calendar($mon, $year)
 
             //If date is today, highlight it
             if (($t == date("j")) && ($mon == date("n")) && ($year == date("Y"))) {
-                echo '<td height="40" valign="top" bgcolor="' . BG_4 . '"><b>' . $t . '</b><br>' . $termin . '</td>';
+                echo '<td height="40" valign="top"><b>' . $t . '</b><br>' . $termin . '</td>';
```
Comment (Pascalmh) [inline calendar.php]: No differentiation anymore...?

**Comment by xxxrenegade** [comment]
Ist das mit dem Installer eigentlich so OK, den bei einem Update geht dann halt der Titel verloren.

**Comment by xxxrenegade** [comment]
Ich denke so sollte es jetzt passen ;-)

File: install/functions.php [hunk 3]
```diff
@@ -2427,6 +2409,8 @@ function update420_430()
     mysqli_query($_database, "INSERT INTO `" . PREFIX . "modrewrite` (`regex`, `link`, `fields`, `replace_regex`, `replace_result`, `rebuild_regex`, `rebuild_result`) VALUES('whoisonline.html#was','index.php?site=whoisonline#was','a:0:{}','index\\\\.php\\\\?site=whoisonline#was','whoisonline.html#was','whoisonline\\\\.html#was','index.php?site=whoisonline#was')");
     mysqli_query($_database, "INSERT INTO `" . PREFIX . "modrewrite` (`regex`, `link`, `fields`, `replace_regex`, `replace_result`, `rebuild_regex`, `rebuild_result`) VALUES('whoisonline/{sort}/{type}.html','index.php?site=whoisonline&sort={sort}&type={type}','a:2:{s:4:\"sort\";s:6:\"string\";s:4:\"type\";s:6:\"string\";}','index\\\\.php\\\\?site=whoisonline[&|&amp;]*sort=(\\\\w*?)[&|&amp;]*type=(\\\\w*?)','whoisonline/$3/$4.html','whoisonline\\\\/(\\\\w*?)\\\\/(\\\\w*?)\\\\.html','index.php?site=whoisonline&sort=$1&type=$2')");
 
+    mysqli_query($_database, "DROP TABLE `" . PREFIX . "settings`");
+
     updateMySQLConfig();
 }
```
Comment (Phhere) [inline install/functions.php]: Das ist nicht ganz korrekt so.
Du darfst die Spalte nicht bei der Basis-Installation hinzufügen sondern nur in der Update-Funktion und musst dann auch den Wert aus der Settings übertragen. Sonst verlieren alle die ein Update machen den Page-Title

Comment (xxxrenegade) [inline install/functions.php]: Das ganze sollte auch eigentlich so lauten:
mysqli_query($_database, "DROP TABLE `" . PREFIX . "styles`");

Allerdings sollte man vor dem löschen  vielleicht den Titel noch auslesen und in die title spalte der settings tabelle einfügen. Oder kann man per SQL auch irgend wie Tabellen Spalten in andere Tabellen verschieben. Ich glaube aber nicht.

File: checklogin.php (hunk 1 above)
Comment (Pascalmh) [inline checklogin.php]: :+1:<a href='#crh-update-none'></a>

File: calendar.php (hunk 2 above)
Comment (Pascalmh) [inline calendar.php]: :+1:<a href='#crh-update-complete'></a>

**Comment by xxxrenegade** [comment]
Jetzt sollte der installer richtig sein ;-)

**Comment by xxxrenegade** [comment]
Jetzt richtig?

**Comment by xxxrenegade** [comment]
So habe jetzt die table-striped Klasse bei den Tabellen hinzugefügt, ist sonst noch etwas was gemacht werden muss? Oder kann es jetzt zusammengeführt werden?

**Comment by Pascalmh** [comment]
Hatten am Dienstag besprochen das wir die Style Seite im ACP doch behalten wollen. Es soll damit dann möglich sein die BS3 Settings zu setzen.

Die wichtigsten Farbänderungen sollten damit möglich sein. http://getbootstrap.com/customize/

Wird können aber hoffentlich teile deines PRs nutzen (löschen der bg1-4 variablen u.A.).

**Comment by xxxrenegade** [comment]
Alles klar, dann konzentriere ich mich mal weiter auf die Anpassung der Einzelnen Module.

**Pull request closed without merging**